zetaclient/evm_signer.go

```diff
@@ -245,6 +245,9 @@ func (signer *EVMSigner) TryProcessOutTx(send *types.CrossChainTx, outTxMan *Out
 			logger.Error().Msgf("Unknown chain: %d", send.GetCurrentOutTxParam().ReceiverChainId)
 			return
 		}
+	} else {
+		logger.Info().Msgf("Transaction doesn't need to be processed status: %d", send.CctxStatus.Status)
```

Comment:
This log line might cause a lot of spam until we fix the zetocored query

Reply:
probably ok as INFO level is restricted in dd. 
also there is a chance that it can resolve itself, depending on which tx was last confirmed
see https://github.com/zeta-chain/node/pull/823